controllers/discovery/initDiscoveryTopicData.js

```diff
@@ -41,10 +43,12 @@ const InitDiscoveryTopicData = async (req, res) => {
       type: QueryTypes.SELECT,
       replacements: {
         userId,
+        anonymousUserId: anonymousUser?.user_id,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `anonymousUserId` is being used as a replacement in the SQL query. As mentioned earlier, ensure that it is properly sanitized to prevent SQL injection attacks.
```diff
  replacements: {
    userId,
+   anonymousUserId: anonymousUser?.user_id,
    limit: limit,
    offset: page * limit
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->